lib/percy.rb

```diff
@@ -77,15 +77,9 @@ def self._make_dom_snapshot(page, options)
 
     begin
       page.execute_script(agent_js)
-      dom_snapshot_js = "new window.PercyAgent({ handleAgentCommunication: false }).domSnapshot(document, #{options.to_json})"
+      dom_snapshot_js = "return new window.PercyAgent({ handleAgentCommunication: false }).domSnapshot(document, #{options.to_json})"
 
-      if self._is_capybara?
-        dom_snapshot = page.evaluate_script(dom_snapshot_js)
-      else
-        dom_snapshot = page.execute_script(dom_snapshot_js)
-      end
-
-      return dom_snapshot
+      page.execute_script(dom_snapshot_js)
```

Comment:
Seems like the snapshot string can be passed directly here now rather than needing a variable to reuse